src/main/java/core/basesyntax/service/CreateBalance.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface CreateBalance {    //most likely, unnecessary
```

Comment:
remove if dont need
or at least redundant comment
```suggestion
public interface CreateBalance {  
```